src/main/resources/application.properties

```diff
@@ -40,6 +37,7 @@ botContext.impl=memoryBotContext
 
 keystores.password=changeit
 keystores.directory=${WON_KEYSTORE_DIR}
+#owner.fixedPrivateKeyAlias=
```

Comment:
why do we comment this, when in fact we dont do it in any other bot (i know that we result to a good default now, but there is no way of ever figuring out how to add a fixedPrivateKeyAlias if the injection happens deep in our bot framework spring config

Reply:
I agree. That was commented out to see if the default works now, forgot to uncomment it.